### PR TITLE
Prepare the release of v0.48

### PR DESCRIPTION
**Pull request by metalmatze**
Time for a new Prometheus Operator release.
@prometheus-operator/prometheus-operator-reviewers

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+## 0.48.0 / 2021-05-19
+
+Deprecation notice:  
+app labels will be removed in v0.50.
+
+* [CHANGE] Replace app label names with app.kubernetes.io/name. #3939
+* [CHANGE] Drop ksonnet as a dependency in jsonnetfile.json. #4002
+* [FEATURE] Reconcile resources on namespace updates when using privileged lister/watcher. #3879
```
Comment (simonpasquier) [inline CHANGELOG.md:8]: BUGFIX rather

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+* [ENHANCEMENT] Add default container annotation to Alertmanager pod. #3978
+* [ENHANCEMENT] Add default container annotation to Thanos ruler pod. #3981
+* [ENHANCEMENT] The prometheus.ConfigGenerator type and its GenerateConfig method are now exported. #3972
```
Comment (simonpasquier) [inline CHANGELOG.md:11]: not sure if it's worth mentioning in the changelog

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,18 @@
...
+* [ENHANCEMENT] Optimize asset secret update logic. #3986
+* [ENHANCEMENT] jsonnet: set default container in prometheus-operator pod. #3979
+* [ENHANCEMENT] Watch configmaps from the Prometheus allowed namespaces only. #3992
```
Comment (simonpasquier) [inline CHANGELOG.md:14]: BUGFIX

File: RELEASE.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ Release cadence of first pre-releases being cut is 6 weeks.
 | v0.45   | 2021-01-13                                 | Lili Cosic (GitHub: @lilic)                 |
 | v0.46   | 2021-02-24                                 | Sergiusz Urbaniak (GitHub: @s-urbaniak)     |
 | v0.47   | 2021-04-07                                 | Simon Pasquier (GitHub: @simonpasquier)     |
-| v0.48   | 2021-05-19                                 | **searching for volunteer**                 |
+| v0.48   | 2021-05-19                                 | Matthias Loibl (GitHub: @metalmatze)        |
 | v0.49   | 2021-06-30                                 | **searching for volunteer**                 |
```
Comment (simonpasquier) [inline RELEASE.md:19]: can we have someone signed up for the next release? @paulfantom?

Comment (simonpasquier) [reply RELEASE.md:19]: If nobody is available, I can volunteer

**Comment by simonpasquier** [comment]
We might want to wait for https://github.com/prometheus-operator/prometheus-operator/pull/4041, we introduced a bug in v0.47 that broke Alertmanager users defining a non-empty `http_config` parameter in the global section.

File: RELEASE.md (hunk 4 above)
Comment (paulfantom) [reply RELEASE.md:19]: I can take it ;)

**Comment by simonpasquier** [comment]
https://github.com/prometheus-operator/prometheus-operator/pull/4041 has merged and can be included in the changelog.

**Comment by paulfantom** [comment]
ping @metalmatze :)

**Review by paulfantom — APPROVED** [review]
LGTM

ping @simonpasquier

**Review by simonpasquier — APPROVED** [review]
lgtm

**Pull request merged**